File: src/dorkbox/vaadin/util/DummyTrustManager.java
package dorkbox.vaadin.util;

import java.security.cert.X509Certificate;

import javax.net.ssl.X509TrustManager;

public
class DummyTrustManager implements X509TrustManager {

    public
    X509Certificate[] getAcceptedIssuers() {
        return new X509Certificate[] {};
    }

    public
    void checkClientTrusted(X509Certificate[] certs, String authType) {
    }

    public
    void checkServerTrusted(X509Certificate[] certs, String authType) {
    }
}

File: src/dorkbox/vaadin/util/DummyTrustManager.kt
package dorkbox.vaadin.util

import java.security.cert.X509Certificate
import javax.net.ssl.X509TrustManager

class DummyTrustManager : X509TrustManager {
    override fun getAcceptedIssuers(): Array<X509Certificate> {
        return arrayOf()
    }

    override fun checkClientTrusted(certs: Array<X509Certificate>, authType: String) {}
    override fun checkServerTrusted(certs: Array<X509Certificate>, authType: String) {}
}
